hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/io/TestHoodieTimelineArchiver.java

```diff
@@ -1221,35 +1222,6 @@ public void testArchiveTableWithMetadataTableCompaction() throws Exception {
         commitsAfterArchival);
   }
 
-  @Test
-  public void testArchiveCommitsWithCompactionCommitInMetadataTableTimeline() throws Exception {
```

Comment:
this test is redundant and uses dummy instants. we're already checking archival with compaction in metadata table in several other tests such as `testArchiveTableWithMetadataTableCompaction` and `testArchivalAndCompactionInMetadataTable`